### PR TITLE
Prepare release 1.1.0

### PR DESCRIPTION
**Pull request by nirarg**
* Change example links
* Add changelog

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: |-
   
 ---
 <a href="https://redhat.com">
-    <img src=".github/Logo_Red_Hat.png" alt="RedHat logo" title="RedHat" align="right" height="50" />
+    <img src="https://raw.githubusercontent.com/terraform-redhat/terraform-provider-rhcs/main/.github/Logo_Red_Hat.png" alt="RedHat logo" title="RedHat" align="right" height="50" />
```
Comment (sagidayan) [inline docs/index.md:9]: Great!

**Review by bardielle — APPROVED** [review]
/approve
/lgtm

**Pull request merged**